src/main/java/com/epam/training/toto/service/TotoService.java

```diff
@@ -1,4 +1,142 @@
 package com.epam.training.toto.service;
 
+import com.epam.training.toto.domain.Outcome;
+import com.epam.training.toto.domain.Round;
+import com.epam.training.toto.service.parser.CSVParser;
+import com.epam.training.toto.service.parser.util.OutcomeParser;
+
+import java.io.IOException;
+import java.text.NumberFormat;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
 public class TotoService {
+    public static final Integer NUMBER_OF_GAMES = 14;
```

Comment:
NUMBER_OF_GAMES is declared two times. Please use separate class Constants with all constants for this project.